omod/src/main/webapp/resources/scripts/diagnoses/diagnoses.js

```diff
@@ -1,10 +1,17 @@
 (function(diagnoses, $, _, undefined) {
-
-    var codingSystemToUse = 'ICD-10-WHO';
+    
+    var defaultPreferredCodingSource = 'ICD-10-WHO';
```

Comment:
Why would this still be here if we introduced `CoreAppsConstants.DEFAULT_CODING_SOURCE`? That's not ok. Somehow this JS should be fed with the default preferred coding source coming from the backend.

Reply:
Sorry about my late response @mks-d, my comment could have disappeared in a draft. I addressed this in the last latest commit.